Tasks/AppCenterDistribute/task.json

```diff
@@ -143,11 +144,14 @@
             "visibleRule": "releaseNotesSelection = file"
         },
         {
-            "name": "distributionGroupId",
+            "name": "destinationId",
```

Comment:
Interesting - you made the _old_ name an alias of the _new_ name. Usually we did the reverse. There's a chance this isn't a breaking change. Do you want to test first, or just proceed with the bump of the major version?

Reply:
Just proceed with the major version bump